kernel/reduction.mli

```diff
@@ -31,7 +31,7 @@ exception NotConvertible
 type 'a kernel_conversion_function = env -> 'a -> 'a -> unit
 type 'a extended_conversion_function =
   ?l2r:bool -> ?reds:TransparentState.t -> env ->
-  ?evars:((existential->constr option) * UGraph.t) ->
+  ?evars:(existential->constr option) ->
```

Comment:
Perhaps a comment to make sure callers remember to set_universes would be a good idea around here.

Reply:
I guess we can do without a comment